modules/zivid/camera.py

```diff
@@ -62,7 +62,7 @@ def capture(self, settings):
         if isinstance(settings, Settings2D):
             return Frame2D(
                 self.__impl.capture(
-                    _settings_2d_converter.to_internal_settings2_d(settings)
+                    _settings2_d_converter.to_internal_settings2_d(settings)
```

Comment:
What forced you to do this? I hate it :O In order of preference: `_settings_2d` >> `_settings_2_d` >>>>> `_settings2_d`.

Reply:
Haha yes, this name is the worst. This function name has been like this since the release of zivid-python 2.0, when Settings2D was created partly by a code-generator. It comes from this:

```
>>> inflection.underscore("Settings2D")
'settings2_d'
```
Note that I did not mess with the module in this PR. I merely changed the import line at the top of the file that was used to locally rename the module.
We have an opportunity to improve these names in my upcoming code-generation PR, since these modules are private, but it seems out of the scope of this PR.

Reply:
I agree, not in scope.